test/test_utils/eks.py

```diff
@@ -215,10 +228,29 @@ def delete_cfn_stack_and_wait(stack_name):
         LOGGER.error(f"Error: Cannot delete stack: {stack_name}. Full Exception:\n{e}")
         describe_cfn_stack_events(stack_name)
 
+def delete_oidc_provider(eks_cluster_name):
+    """Function to delete the oidc provider created by kubeflow
+    Args:
+        eks_cluster_name: str
+    """
+    eks_client = init_eks_client()
+    iam_client = init_iam_client()
+    account_id = os.getenv("ACCOUNT_ID")    
```

Comment:
Could you please use boto3 APIs to find the account ID instead?

https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sts.html#STS.Client.get_caller_identity

We can never be 100% sure that an environment variable will always be populated unless it is a "necessary" value for a previous section in the same workflow. It's always best to have a default value for all `getenv()` calls. In this case, we don't actually need environment variables as the account ID can be identified through boto3 APIs.

Reply:
uses boto3 sts client